src/nvim/os/fs.c

```diff
@@ -120,9 +120,13 @@ static bool is_executable(const char_u *name)
     return false;
   }
 
-  if (S_ISREG(mode) && (S_IXUSR & mode)) {
-    return true;
-  }
+#if WIN32
+  // Windows does not have exec bit, just check if
+  // the file exists
+  return (S_ISREG(mode));
```

Comment:
Could do this instead:

    return os_access(name, F_OK);

Then `os_getperm` is not needed for the WIN32 branch. This will require defining F_OK in win_defs.h:

    #  define F_OK 0

See https://github.com/justinmk/neovim/commit/b1e18dc39325ef50121a28750463a068048d63cb